src/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  posts: Post[];
+};
+
+const PostsList: React.FC<Props> = ({ posts }) => (
+  <ul>
+    {posts.map((post) => (
```

Comment:
Nothing wrong here, but I have a suggestion for improvement. Consider adding a unique key prop to the <li> element when mapping over the posts array. This aids React in efficiently updating and re-rendering the list when changes occur. Check it out:

 ```
{posts.map((post) => (
      <li key={post.id}> {/* Add key prop here */}
        <b>{post.title}</b>
        <div>{post.body}</div>
        <hr />
      </li>
    ))}


```